geniza/corpus/views.py

```diff
@@ -19,6 +19,19 @@ def get_queryset(self, *args, **kwargs):
         return queryset.filter(status=Document.PUBLIC)
 
 
+class DocumentScholarshipView(DocumentDetailView):
+    """List of :class:`~geniza.footnotes.models.Footnote`s for a Document"""
+
+    template_name = "corpus/document_scholarship.html"
+
+    def get_queryset(self, *args, **kwargs):
+        """Don't show document if there are no footnotes"""
+        # prefetch footnotes since we'll render all of them in the template
+        queryset = super().get_queryset(*args, **kwargs)\
+                          .prefetch_related("footnotes")
```

Comment:
this needs a `.distinct()` added — I got an error when I tried to load this page for a document with two footnotes.

Also should filter on document status set to public

Reply:
Inheriting from `DocumentDetailView` should already filter on status, no? That's what I was hoping it would do

Reply:
oh, I missed that ! yes, perfect

Reply:
Also, wouldn't `distinct()` just operate on the level of the document? `DetailView`s should only ever return one object in the queryset otherwise you'd get `MultipleObjectsReturned`, I think

Reply:
you can get multiple rows if there are multiple footnotes — the distinct is to collapse them back to a single document. multiple objects returned was exactly what I got for the document with two footnotes